src/main/java/org/eclipse/golo/doc/FunctionDocumentation.java

```diff
@@ -170,25 +170,50 @@ public FunctionDocumentation parent(DocumentationElement p) {
 
   @Override
   public int compareTo(DocumentationElement other) {
-    if (this == other) { return 0; }
-    if (null == other) { return 1; }
+    if (this == other) {
+      return 0;
+    }
+    if (null == other) {
+      return 1;
+    }
     if (other instanceof FunctionDocumentation) {
       FunctionDocumentation o = (FunctionDocumentation) other;
       int c = name.compareToIgnoreCase(o.name);
       if (c == 0) {
         c = arity() < o.arity() ? -1 : 1;
-        if (c == 0) {
-          c = varargs && !o.varargs() ? 1
-              : varargs && o.varargs() ? 0
-              : -1;
-          if (c == 0) {
-            c = parent.compareTo(o.parent);
-          }
-        }
       }
```

Comment:
Not sure here... Indeed it's dead code, but may be due to a bug. We are comparing functions. If arities are the same, we indeed should take varargs into account, no? I think the error is in the ternary operator comparing arity.